include/seqan3/alignment/configuration/detail.hpp

```diff
@@ -22,8 +22,8 @@ namespace seqan3::detail
  */
 enum struct align_config_id : uint8_t
 {
-    //!\brief ID for the \ref seqan3::align_cfg::alignment_result_capture "alignment_result_capture" option.
-    alignment_result_capture,
+    //!\brief ID for the \ref seqan3::align_config::detail::result_type "result_type" option.
```

Comment:
Should be 
```suggestion
    //!\brief ID for the \ref seqan3::align_cfg::detail::result_type "result_type" option.
```

Reply:
I think its supposed to be like this. In the belonging issue https://github.com/seqan/product_backlog/issues/183 it looks like this namespace `seqan3::align_cfg::detail` should be part of the public interface. I'll assign @rrahn as second reviewer, he wrote the issue.